backend/src/libs/exceptions/database/database-connection.exception.ts

```diff
@@ -1,7 +1,4 @@
-type Constructor = {
-  message: string;
-  cause?: unknown;
-};
+import { type Constructor } from '~/libs/types/types.js';
```

Comment:
I had a custom exception, and to avoid code duplication, I moved the type to a separate file. However, during a code refactor, I deleted the custom exception but retained the type.